### ISSUE TITLE
Regaining access to hamster organisation - maybe move to new organsation?

### ISSUE THREAD

**Issue by matthijskooijman** [projecthamster/hamster#730]
This is a followup for a discussion that recently started at https://github.com/projecthamster/hamster-shell-extension/issues/362#issuecomment-1735364629

The projecthamster organization is owned by @elbenfreund. A few years ago, he has added a number of collaborators to the "legacy" (i.e. not the in-progress-but-stalled rewrite that @elbenfreund was working on) repositories, allowing us to continue maintaining the hamster code. However, we do not have full admin access to the repositories and organization, which means we cannot manage some things ourselves. In particular, the following things have come up over the years:
 - We want to [add a new maintainer](https://github.com/projecthamster/hamster/issues/574) to some repositories
 - We want to [add a new repository](https://github.com/projecthamster/hamster/issues/719) for the KDE applet.
 - We want to [change protection rules](https://github.com/projecthamster/hamster-shell-extension/issues/362#issuecomment-1735256019) for some branches
 - We want to change the name of the "hamster-legacy" team

We have tried to contact @elbenfreund about either making these changes, or giving out more access so we can manage these things ourselves. I have personally send him e-mails about this, first in 2019, a reminder in 2021, and once more in April 2023 (the last time including all e-mail addresses I could find online), but have received zero response to them. I believe others have also made attempts to contact @elbenfreund with the same result.

### Options forward
 1. Ideally, @elbenfreund would become responsive again and would share his admin access to the organisation. Just access to the repositories is not enough (then we cannot add new repositories). However, if @elbenfreund would commit to making such changes on our request in the future without giving out admin access, I would (personally) not consider that sufficient, given his unresponsiveness in the past.
 2. We could create a new organization (I would suggest naming it hamster-time-tracker, to prevent name-squatting, I've gone ahead and [claimed the name already](https://github.com/hamster-time-tracker)). This is an easy way that we can implement right away, but has a number of downsides: External links must be updated, we lose issue and pullrequest history, users might be confused (since we can add pointers to the repos, but not the organisation info).
 3. We could try contacting Github support and see if they would want to give us access. I have no idea if this is feasible at all ([this FAQ suggests not](https://docs.github.com/en/site-policy/other-site-policies/github-account-recovery-policy#can-i-recover-the-contents-of-a-user-or-organization-account-i-lost-access-to), but it does mitigate all the downsides of the previous option. It has its own downsides: This will likely take some time and effort on our part, and is potentially quite impolite towards @elbenfreund (but that can probably be mitigated by a proper process, see below).

### Process
Since we're effectively considering bypassing @elbenfreund, I believe we should do this carefully. I can image we would:
 1. Figure out which of option 2 and 3 above we would prefer (assuming that option 1 is of course the ideal option, but also assuming it is not obtainable).
 2. Give @elbenfreund a notice of our plans, and some time (say 4 weeks or so?) to respond and discuss or implement option 1.
 3. Lacking any response, move forward with implementing the plan.

Alternatively, we could skip step 1 and start with step 2, suggesting that we're planning to take steps to get access again without specifying which exactly.

**Comment by matthijskooijman** [comment]
A few (excerpts from) comments from https://github.com/projecthamster/hamster-shell-extension/issues/362:

@benjaoming [said](https://github.com/projecthamster/hamster-shell-extension/issues/362#issuecomment-1735461198):

> This is always a huge problem, not to mention that the extension doesn't even have a proper official package on Gnome. I've followed the conversation and all the issues over the years and I could totally get behind a reboot, mainly in terms of governance and distribution. I think renaming the project[1] would help to free up various namespaces in the distribution chain.

> [1] for instance to a similar animal that likes running around in wheels, example: "chinchilla"

@mwilck [said](https://github.com/projecthamster/hamster-shell-extension/issues/362#issuecomment-1735549931):

> About changing the name: I'm unsure if that's a good idea. People will be looking for "hamster", both on e.g.o and elsewhere. AFAICS, nobody owns the name "hamster", and if anyone did, it would probably be @tstriker. IMO we could simply fork the active repos and keep using the name "hamster", as long as we keep true to the spirit of the project (which I think we have done so far).

@tstriker [said](https://github.com/projecthamster/hamster-shell-extension/issues/362#issuecomment-1735678551):

> the name 'hamster' ('project hamster', specifically) is not copyrighted and nobody owns it so please keep using it if you want - the name was a reference to the hamster wheel and is a tongue-in-cheek reference to 9-to-5 work.
> 
> if elbenfreund has gone missing, you might want to reach out to github support - they might allow transferring the project ownership to a new team. until then you can also just abandon 'master' as github deprecated those in favor to 'main' a while ago anyway.
> 
> alternatively, of course, you could make a fork org and leave a message somewhere pointing to it but that might be messier.

@mwilck [said](https://github.com/projecthamster/hamster-shell-extension/issues/362#issuecomment-1735835592):

> Wrt reaching out to github support, I see that as a last resort. Even if @elbenfreund has turned silent, his contributions are massive and still very valuable, and I wouldn't want to be rude to him. I'd strongly appreciate if he voluntarily assigned someone else (one or more of the currently active people) a maintainer role in this repository though. If he can't or doesn't want to do that, I'd rather move to a fork. That's my personal PoV.

**Comment by benjaoming** [comment]
Great work @matthijskooijman, thanks for gathering this overview - I guess this discussion in itself requires time. But that's great, if it can help forge a new community with strong alignment :heart: 

**a) Staying in this organization:** All the legacy of old discussions and confusing permissions and zero governance will just remain. This cannot be fixed IMO.

**b) A new organization, under the same name:** This is a better idea, will give active maintainers a chance to do things the right way.

**c) A new organization, new name:** Clean up all confusion around what Hamster is and where to find original sources and packages.

Id' be most happy with option c) but I'm not saying that option b) is unacceptable, I think it will fix a lot of issues.

I'm not gonna run out and launch a renamed fork tomorrow :blush: but I would love if we were 4-5 people that create a new maintainer group, plan our actions on a Kanban board and have a renamed version of the project ready in the next 3-6 months.

**Comment by WBTMagnum** [comment]
@matthijskooijman Thanks for the initiative, the sum-up and the work you put into this.

One thought: There's also the option to give @elbenfreund a call (there are at least two numbers available online) or reach out via LinkedIn. If he doesn't follow his Github notifications and E-Mails anymore, that might be an option to make him aware of the situation, so this can hopefully be cleared up - in whatever way - quickly.

**Comment by matthijskooijman** [comment]
> One thought: There's also the option to give @elbenfreund a call (there are at least two numbers available online) or reach out via LinkedIn.

Good point - I've sent him a connection request through linkedin just now, let's see if that works. Trying a call is also something I'd be willing to try.

**Comment by GeraldJansen** [comment]
I am very supportive and would be willing to help with a "quick" move to a new organization. The name hamster-time-tracker is fine. There should be at least 3 people with top-level permissions so the project is less likely to get stuck again. 

Note that beside hamster and hamster-shell-extension, xfce4-hamster-plugin is also maintained and in perfect working order (thanks to @aquaherd). Also, Debian packager @rhertzog has offered to participate as a hamster maintainer in the past and may still be willing.

**Comment by kde-baskets-user** [comment]
Hello. any progress on this 1 year later?
